Include/PGE/String/String.h

```diff
@@ -287,47 +290,68 @@ class String {
         bool isEmpty() const;
 
     private:
-        String(int size);
+        String(nullptr_t) { } // Dummy constructor
```

Comment:
Default constructor seems cheap enough, I would remove this altogether

Reply:
If we were to use that one we'd have to manually re-invalidate the metadata.